src/Pages/3DPrinting/3DPrintForm.js

```diff
@@ -116,6 +128,22 @@ export default class PrintForm3D extends React.Component {
 
   render () {
     const { fill } = this.state
+    const printingColors = [
+      'Select Color',
```

Comment:
We should make ```Any Color``` as default

Reply:
fixed. also removed code for checking if color input is valid since default value is always valid